xilem_web/src/lib.rs

```diff
@@ -1,6 +1,23 @@
 // Copyright 2023 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
+#![doc = concat!(
+"
+<!-- This license link is in a .rustdoc-hidden section, but we may as well give the correct link -->
+[LICENSE]: https://github.com/linebender/xilem/blob/main/xilem_web/LICENSE
+
+<!-- intra-doc-links go here -->
+
+<style>
+.rustdoc-hidden { display: none; }
+</style>
+
+<!-- Hide the header section of the README when using rustdoc -->
+<div style=\"display:none\">
```

Comment:
This format of the README is out-of-date as compared to https://github.com/linebender/linebender.github.io/pull/55

But I'll update both of them later, so this is still fine